index.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
 				'unicorn/filename-case': ['error', {case: 'kebabCase'}],
 				'unicorn/no-abusive-eslint-disable': 'error',
 				'unicorn/no-process-exit': 'error',
+				'unicorn/no-unreadable-array-destructuring': 'error',
```

Comment:
Should be added to the end. https://github.com/sindresorhus/eslint-plugin-unicorn/pull/198#discussion_r228740711